src/util/TestCaseExcelExportUtil.ts

```diff
@@ -129,23 +129,22 @@ export const createExcelExportDtosForAllTestCases = (
     const testCaseExecutionResultDtos: TestCaseExecutionResultDto[] = [];
     const testCasesByGroup: TestCase[] = getTestCasesByGroup(measure, group.id);
     testCasesByGroup?.forEach((currentTestCase) => {
-      const groupPopulations = currentTestCase.groupPopulations?.filter(
+      const groupPopulation = currentTestCase.groupPopulations?.filter(
         (groupPopulation) => {
           return groupPopulation.groupId === group.id;
         }
       );
       const populationDtos: PopulationDto[] = [];
       let groupedStratDtos: GroupedStratificationDto[] = [];
       let stratNumber = 1;
-      groupPopulations?.forEach((groupPopulation) => {
-        populatePopulationDtos(groupPopulation, populationDtos);
-        groupedStratDtos = populateStratificationDtos(
-          groupPopulation,
-          groupNumber,
-          stratNumber,
-          currentTestCase.id
-        );
-      });
+
+      populatePopulationDtos(groupPopulation[0], populationDtos);
```

Comment:
Do you think we could update this method to return **populationDtos** rather than passing the variable reference ?  which could be difficult to debug.

Reply:
@RohitKandimalla Changed.